app/modules/ads/setup.js

```diff
@@ -78,10 +79,12 @@ function setupAdContext(adsContext, instantGlobals) {
 	slots.setupStates();
 }
 
-function configure(adsContext, instantGlobals) {
+function configure(adsContext, instantGlobals, isOptedIn) {
 	const {context} = window.Wikia.adEngine;
+	const {utils: adProductsUtils} = window.Wikia.adProducts;
```

Comment:
const adProductsUtils = window.Wikia.adProducts.utils;
looks more readable IMO

Reply:
I agree, but this convention allows us to switch to imports easily – it's a temporary solution.